src/components/ViewWrapper.vue

```diff
@@ -390,8 +355,11 @@ export default class ViewWrapper extends Mixins(CommonMixin, FilingTemplateMixin
       } else {
         const error = new Error('Missing Payment Token or Filing ID')
         this.$root.$emit('save-error-event', error)
+        this.setIsSaving(false)
```

Comment:
Not having this was actually a bug -- the isSaving state was left as True if this error occurred, leaving the user unable to client any buttons.